src/cli/mod.rs

```diff
@@ -190,6 +191,30 @@ impl Opt {
     }
 }
 
+fn get_and_migrate_dir<P: AsRef<Path> + PartialEq>(
```

Comment:
I would prefer not to actually move the directories. I think this could be a bit invasive. I'd rather just log a warning and link to the release notes where we can document what options people have.

if we do add something like this, I think it should be its own subcommand and I would prefer that be done in a separate PR.

Reply:
We're only moving the directory if they were using the default dir before and are using the default dir afterwards. Moving is less invasive than breaking IMO, but you're right that doing so silently might be problematic. What do you think about logging a warning and moving? As for a separate subcommand: I think that'd just make it even more complicated, and we should probably avoid that.